src/rulesets/packages/SpecialFunctions.jl

```diff
@@ -1,3 +1,8 @@
+module SpecialFunctionsGlue
```

Comment:
Is `SpecialFunctionsGlue` the best name for this? Perhaps `SpecialFunctionsRules` not more appropriate?

Reply:
One of the best things about making thes internal is not needing to stress too much over the name.
I'm inclide to leave it as is.
It is an awkward thing so gets an awkward name